### ISSUE TITLE
Prevent tab autoplay/one instance playing buggy with FF 53 beta 1

### ISSUE THREAD

**Issue by kilara1988** [YePpHa/YouTubeCenter#2388]
My FF just updated to FF53 beta 1 and i started experiencing some bugs with YTC.
I have the prevent tab auto play option enabled, but almost always the video in the background tab plays for just half a second before it stops.Sometimes it doesn't even stop and continues playing.
One time i had a video playing on my active tab and opened a new video in a background tab and both videos were playing at the same time.So it seems that the only one play instance playing option is not working always too.

At first i thought it might be because of the newly introduced feature "out of process GPU rendering" or something like that which now spawns a new separate process for some GPU related stuff.I disabled it,but the issues remained.

Also now much more often when i scroll up to use the Fullscreen Top Player option the player just disappears and leaves the white background of the page while the video still plays.

Using script with FF 53 Beta 1 on Win 10 64bit.

**Comment by Nicholas-Steel** [comment]
This URL https://www.youtube.com/watch?v=KmpmsNBpUEY (via https://www.youtube.com/user/ThatOneLaserClown/videos?view=0&flow=list ) always auto-plays even though all auto-play functionality is disabled via YTC.

https://www.youtube.com/watch?v=VaWf3C4TAak (via https://www.youtube.com/user/GameGrumps/videos?view=0&flow=list ) auto-plays when I clicked the link to be taken to the page, but refreshing the page correctly results in the video not auto-playing.

Firefox 53 beta 2 x64.

**Comment by Nicholas-Steel** [comment]
Seems this might, maybe be fixed in Firefox 53 Beta 4.

**Comment by Nicholas-Steel** [comment]
Nope, lots of broken stuff with Firefox 53 in general. None of the new betas since "53 Beta 1" have fixed the various issues plaguing Youtube Center. Generally a refresh of the page gets everything working (at least the components of Youtube Center that I utilize seem to work)

**Comment by kilara1988** [comment]
Firefox 53 just landed today and i tested if the script is still broken with FF 53.
Sadly it is still broken just as before.Background tabs still play for a half second before stopping and they still auto stop playing if you switch out of that tab with the video and so on.

I will stay on FF 52 for now in hopes of a fix,but if it doesn't get fixed soon i will have to switch to another less feature rich addon/script :(
